### ISSUE TITLE
ngContextMenu inside uiGrid's row template wrong posotion

### ISSUE THREAD

**Issue by ndjordjevic** [Wildhoney/ngContextMenu#22]
ngContextMenu calculates coordinates wrongly when is placed inside of row template in UIGrid even with position:absolute. I think it takes grid's coordinates instead screen's.

**Comment by Wildhoney** [comment]
Yep &ndash; `absolute` is a whole different scenario, as it's relative to the UIGrid, so although it gets the correct coodinates, the values aren't relative to its container.

I'll have to think about that, as `ngContextMenu` is designed to work with `position: fixed` which doesn't suffer from the same complexities as `absolute`.

**Comment by bendspoons** [comment]
I just made a workaround to fix the Position of the ContextMenu by adding "$(window).scrollTop()" to the y Pos (in Line 137, uncompressed file).
